packages/clusview/samplers/clusters/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""
+Functions for clustering algorithms.
```

Comment:
Yes, but these are samplers that use those algorithms. The `HDBSCAN` component here is a sampler of the algorithms. Meaning that it samples over different configurations of the algorithm parameters.